http/check_service.go

```diff
@@ -528,7 +529,6 @@ func (h *CheckHandler) handlePostCheck(w http.ResponseWriter, r *http.Request) {
 		h.HandleHTTPError(ctx, err, w)
 		return
 	}
-	h.log.Debug("Check created", zap.String("check", fmt.Sprint(chk)))
```

Comment:
i think this is still being used. Am I right, @gshif?

Reply:
@kelwang `fmt.Sprint`ing the raw check is maybe not the thing to do though. This can easily leak sensitive information like the secrets on a check.